Indicators/KaufmanAdaptiveMovingAverage.cs

```diff
@@ -32,6 +32,8 @@ public class KaufmanAdaptiveMovingAverage : WindowIndicator<IndicatorDataPoint>,
         private decimal _prevKama;
         private decimal _trailingValue;
 
+        public IndicatorBase<IndicatorDataPoint> KER { get; }
```

Comment:
I suggest calling it `EfficiencyRatio` because `KER` might be obscure.
Also, the XML comments are missing and they are important for documentation.